trie/trieStorageManager.go

```diff
@@ -314,6 +314,25 @@ func (tsm *trieStorageManager) Get(key []byte) ([]byte, error) {
 	return tsm.getFromOtherStorers(key)
 }
 
+//GetFromCurrentEpoch checks only the current storer for the given key, and returns it if it is found
+func (tsm *trieStorageManager) GetFromCurrentEpoch(key []byte) ([]byte, error) {
+	tsm.storageOperationMutex.Lock()
```

Comment:
I would keep the mutex only to get the storer pointer. 
Otherwise, all GetFromCurrentEpoch will be serialized.

Reply:
Done.